deps/rabbitmq_mqtt/test/disconnect_on_unauthorized_SUITE.erl

```diff
@@ -0,0 +1,141 @@
+%% This Source Code Form is subject to the terms of the Mozilla Public
+%% License, v. 2.0. If a copy of the MPL was not distributed with this
+%% file, You can obtain one at https://mozilla.org/MPL/2.0/.
+%%
+%% Copyright (c) 2007-2025 Broadcom. All Rights Reserved. The term “Broadcom” refers to Broadcom Inc. and/or its subsidiaries. All rights reserved.
+
+
+-module(disconnect_on_unauthorized_SUITE).

```

Comment:
You probably need to add this suite to the Makefile.

Reply:
Let's add it to `PARALLEL_CT_SET_1_A` in  https://github.com/rabbitmq/rabbitmq-server/blob/dd9633976d2b550d6ee21e95647bfde2f5be8cc7/deps/rabbitmq_mqtt/Makefile#L98-L102